manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.8/MuhammadFareezIqmal.IIUMSchedule.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: MuhammadFareezIqmal.IIUMSchedule
+PackageVersion: 0.1.0.8

```

```suggestion
PackageVersion: 0.1.0.7
```
This must match Control Panel.

---

Ahh. How can I miss this one. I'll reupload a new installer with updated version. Is that okay?

---

Sure.